cmd/eksctl-anywhere/cmd/deletecluster.go

```diff
@@ -98,7 +98,7 @@ func (dc *deleteClusterOptions) deleteCluster(ctx context.Context) error {
 		WithBootstrapper().
 		WithClusterManager(clusterSpec.Cluster).
 		WithProvider(dc.fileName, clusterSpec.Cluster, cc.skipIpCheck, dc.hardwareFileName, cc.skipPowerActions, cc.setupTinkerbell, false).
-		WithFluxAddonClient(clusterSpec.Cluster, clusterSpec.FluxConfig).
+		WithFluxAddonClient(clusterSpec.Cluster, clusterSpec.FluxConfig, nil).
```

Comment:
If this is nil everywhere else except for create, can we make this an option param? I did it recently here: https://github.com/aws/eks-anywhere/blob/main/pkg/git/factory/gitfactory.go#L23

Reply:
But thinking about, might be overkill if we are only setting it in the constructor and not doing anything else with it. I am not sold on this.

Reply:
I'm going to pass a CLI config for these

Reply:
updated to include the config here